rero_ils/modules/libraries/tasks.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Tasks on `Library` resource."""
+
+from celery import shared_task
+from invenio_cache import current_cache
+
+
+@shared_task(bind=True, name='library-calendar-changes-update-loans')
+def calendar_changes_update_loans(self, record_data):
```

Comment:
The documentation of python says:
The `self` parameter is a reference to the current instance of the class, and is used to access variables that belongs to the class.

This is not a `class` and I would prefer not to use `self` here.

Reply:
I didn't need the task reference (previously I check the task_id) so I can remove the `self` reference.
But using the `@shared_task(bind=True)`, it seems convenient/usage to use `self` as first argument.